languagetool-language-modules/es/src/main/resources/org/languagetool/resource/es/multiwords.txt

```diff
@@ -3472,283 +3471,283 @@ Zorita de la Frontera;NPCSG00 #municipios de España
-San Juan Nepomuceno;NPCSG00 #municipios de Colombia
-San Juanito;NPCSG00 #municipios de Colombia
-San Lorenzo;NPCSG00 #municipios de Colombia
-San Luis;NPCSG00 #municipios de Colombia
-San Luis de Gaceno;NPCSG00 #municipios de Colombia
-San Luis de Sincé;NPCSG00 #municipios de Colombia
-San Marcos;NPCSG00 #municipios de Colombia
-San Martín;NPCSG00 #municipios de Colombia
-San Martín de Loba;NPCSG00 #municipios de Colombia
-San Mateo;NPCSG00 #municipios de Colombia
-San Miguel;NPCSG00 #municipios de Colombia
-San Miguel de Sema;NPCSG00 #municipios de Colombia
-San Onofre;NPCSG00 #municipios de Colombia
-San Pablo;NPCSG00 #municipios de Colombia
-San Pablo de Borbur;NPCSG00 #municipios de Colombia
-San Pedro;NPCSG00 #municipios de Colombia
-San Pedro de Cartago;NPCSG00 #municipios de Colombia
-San Pedro de Uraba;NPCSG00 #municipios de Colombia
-San Pelayo;NPCSG00 #municipios de Colombia
-San Rafael;NPCSG00 #municipios de Colombia
-San Roque;NPCSG00 #municipios de Colombia
-San Sebastián;NPCSG00 #municipios de Colombia
-San Sebastián de Buenavista;NPCSG00 #municipios de Colombia
-San Vicente;NPCSG00 #municipios de Colombia
-San Vicente de Chucurí;NPCSG00 #municipios de Colombia
-San Vicente del Caguán;NPCSG00 #municipios de Colombia
-San Zenón;NPCSG00 #municipios de Colombia
-Santa Ana;NPCSG00 #municipios de Colombia
-Santa Bárbara;NPCSG00 #municipios de Colombia
-Santa Bárbara de Pinto;NPCSG00 #municipios de Colombia
-Santa Catalina;NPCSG00 #municipios de Colombia
-Santa Helena del Opón;NPCSG00 #municipios de Colombia
-Santa Isabel;NPCSG00 #municipios de Colombia
-Santa Lucía;NPCSG00 #municipios de Colombia
-Santa María;NPCSG00 #municipios de Colombia
-Santa Marta;NPCSG00 #municipios de Colombia
-Santa Rosa;NPCSG00 #municipios de Colombia
-Santa Rosa de Cabal;NPCSG00 #municipios de Colombia
-Santa Rosa de Osos;NPCSG00 #municipios de Colombia
-Santa Rosa de Viterbo;NPCSG00 #municipios de Colombia
-Santa Rosa del Sur;NPCSG00 #municipios de Colombia
-Santa Rosalía;NPCSG00 #municipios de Colombia
-Santa Sofía;NPCSG00 #municipios de Colombia
-Santafé de Antioquia;NPCSG00 #municipios de Colombia
-Santander de Quilichao;NPCSG00 #municipios de Colombia
-Santiago de Tolú;NPCSG00 #municipios de Colombia
-Santo Tomás;NPCSG00 #municipios de Colombia
-Talaigua Nuevo;NPCSG00 #municipios de Colombia
-Tolú Viejo;NPCSG00 #municipios de Colombia
-Unión Panamericana;NPCSG00 #municipios de Colombia
-Valle de Guamez;NPCSG00 #municipios de Colombia
-Valle de San José;NPCSG00 #municipios de Colombia
-Valle de San Juan;NPCSG00 #municipios de Colombia
-Vigía del Fuerte;NPCSG00 #municipios de Colombia
-Villa Caro;NPFSG00 #municipios de Colombia
-Villa de Leyva;NPFSG00 #municipios de Colombia
-Villa de San Diego de Ubate;NPFSG00 #municipios de Colombia
-Villa del Rosario;NPFSG00 #municipios de Colombia
-Villa Rica;NPFSG00 #municipios de Colombia
-Vista Hermosa;NPCSG00 #municipios de Colombia
-Zona Bananera;NPCSG00 #municipios de Colombia
+Agua de Dios;NPCSG00
```

Comment:
the comment is removed?

Reply:
Oh, no!! I picked the wrong file 😢

Reply:
It should be solved now. Thank you for the heads-up!